src/html/tags/HtmlTagSpan.js

```diff
@@ -1,24 +1,45 @@
 /* @flow */
 import React, { PureComponent } from 'react';
 import { View } from 'react-native';
+import { connect } from 'react-redux';
 
 import type { Actions, StyleObj } from '../../types';
+import { getOwnEmail } from '../../selectors';
 import renderHtmlChildren from '../renderHtmlChildren';
+import styles from '../HtmlStyles';
 
-export default class HtmlTagSpan extends PureComponent {
+class HtmlTagSpan extends PureComponent {
   props: {
     style?: StyleObj,
     actions?: Actions,
     cascadingStyle?: StyleObj,
+    attribs?: Object,
+    ownEmail: string,
+  };
+
+  mentionButtonRefersToSelf = () => {
+    const { attribs, ownEmail } = this.props;
+    return attribs && attribs.class === 'user-mention' && attribs['data-user-email'] === ownEmail;
```

Comment:
Considering that the mention-self span already have a class called 'user-mention-me', this is likely too much complication.